src/docs/content/disaster-recovery/setup.md

```diff
@@ -0,0 +1,273 @@
+    spec:
+    tls:
+    - secretName: tls-secret
+    rules:
+    - http:
+        paths:
+        - backend:
+            serviceName: edge
+            servicePort: 80
+    EOF
+    ```
+
+- Access the Source MuShop application using the ingress IP
+
+   ```bash
+   kubectl get svc mushop-utils-ingress-nginx-controller \
+  --namespace mushop-utilities
+   ```
+- Verify the application at Source
+
+   Access `https://Ingress-IP-Address` and ensure that you would see the all the MuShop catalogue products listed without errors.
+
+### Perform Autonomous Transaction Processing (ATP) Failover
+
+Go to OCI console and perform a failover.
+``` text
+OCI-Console -> Oracle Database -> Autonomous Transaction Processing (Standby db: ap-hyderabad-1) -> Switchover
+```
+
+{{% alert style="information" icon="information" %}}
+Wait till the switchover completes fully and there are no 'role change in progress' status on either side (Primary and Standby). 
+{{% /alert %}}
+
+### MuShop Setup (Disaster Recovery (DR) site ap-hyderabad-1)
+
+Change your OKE cluster to DR. If you dont have a DR OKE cluster setup yet then refer to the [Create OKE clusters](#create-oke-oracle-cloud-infrastructure-container-engine-for-kubernetes-clusters) section and create a OKE cluster at the DR region.
+
+- Download and extract the DR ADB wallet
+
+```text
+OCI-Console -> Oracle Database -> Autonomous Transaction Processing (Standby db: ap-hyderabad-1) -> DB Connection -> Download wallet
+```
+
+- Create the secrets, set the region as ap-hyderabad-1 in this case
+
+    ```bash
+    kubectl create secret generic oci-credentials \
+    --namespace mushop \
+    --from-literal=tenancy=<TENANCY_OCID> \
+    --from-literal=user=<USER_OCID> \
+    --from-literal=region=<USER_OCI_REGION> \
+    --from-literal=fingerprint=<USER_PUBLIC_API_KEY_FINGERPRINT> \
+    --from-literal=passphrase=<PASSPHRASE_STRING> \
+    --from-file=privatekey=<PATH_OF_USER_PRIVATE_API_KEY>
+    ```
+
+    ```bash
+    kubectl create secret generic oadb-wallet   --namespace mushop   --from-file=/tmp/wallet_remote
+    ```
+    ```bash
+    kubectl create secret generic oadb-admin \
+    --namespace mushop \
+    --from-literal=oadb_admin_pw='DemoAdb#1234'
+    ```
+    ```bash
+    kubectl create secret generic oadb-connection \
+    --namespace mushop \
+    --from-literal=oadb_wallet_pw='DemoAdb#1234' \
+    --from-literal=oadb_service='demoadb_tp'
+    ```
+- Edit/Add the following secrets to values-prod.yaml as shown below
+
+    ```bash
+    cat mushop/values-prod.yaml
+    global:
+    ociAuthSecret: oci-credentials        # OCI authentication credentials secret name
+    ossStreamSecret:                      # Name of Stream Connection secret
+    oadbAdminSecret: oadb-admin           # Name of DB Admin secret created earlier
+    oadbWalletSecret: oadb-wallet         # Name of wallet secret created earlier
+    oadbConnectionSecret: oadb-connection # Name of connection secret created earlier
```

Comment:
Separate the actual command that the user can copy from the example result from the command